c_binding/libsg.c

```diff
@@ -803,3 +813,130 @@ int _sg_tp_sas_addr_of_disk(char *err_msg, int fd, char *tp_sas_addr)
 
     return rc;
 }
+
+int _sg_io_mode_sense(char *err_msg, int fd, uint8_t page_code,
```

Comment:
Is MODE SENSE (6) specifically unsupported here? When I implemented this on my end for SMART Status, I actually implemented both and had the ability to try both. If we don't need 6 I'm ok with not adding it, but curious if you know one way or the other.